### PR TITLE
Added fix for https://github.com/hapifhir/hapi-fhir-jpaserver-starter…

### PR DESCRIPTION
**Pull request by jkiddo**
…/issues/328

### CODE REVIEW

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java [hunk 1]
```diff
@@ -277,6 +277,13 @@ private void init600() {
 		version
 			.onTable("HFJ_RESOURCE")
 			.dropIndex("20220314.1", "IDX_INDEXSTATUS");
+
```
Comment (jamesagnew) [inline hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java:280]: Could you move these to the bottom of the method and give them today's date? (Flyway complains if new tasks get inserted before previously existing tasks) Happy to merge with that done.

Comment (jkiddo) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/migrate/tasks/HapiFhirJpaMigrationTasks.java]: @jamesagnew should be good to go

**Comment by tadgh** [comment]
Hey @jkiddo , can you resolve the merge conflicts and re-push? I enabled auto-merge for once they're resolved

**Comment by jkiddo** [comment]
> Hey @jkiddo , can you resolve the merge conflicts and re-push? I enabled auto-merge for once they're resolved

done

**Comment by jkiddo** [comment]
@jamesagnew / @tadgh it should be good to go now

**Review by jamesagnew — APPROVED** [review]

**Pull request merged**